lib/ui/viewer/search/search_widget.dart

```diff
@@ -76,7 +81,8 @@ class _SearchWidgetState extends State<SearchWidget> {
                       keyboardType: TextInputType.visiblePassword,
                       // Above parameters are to disable auto-suggestion
                       decoration: InputDecoration(
-                        hintText: 'Search for albums, places, holidays & years',
+                        hintText:
+                            'Search for albums, places, holidays, months & years',
```

Comment:
@mnvr approve this string?

Reply:
"Places, moments, albums..." 

(include the ellipses at the end, although the text field might also put its own ellipses earlier too on smaller devices)

---

Copy for no results:

(center aligned) No results found 

(left aligned) You can try using a different query. ente can find results for:

* Places (e.g. "London")
* Years and months (e.g. "2022", "January")
* Holidays (e.g. "Christmas")
* Album names (e.g. "Recents")

Reply:
Note: **ente** will be bold

Reply:
These copy changes can happen in a subsequent PR, I see that this PR is approved so can merge this instead of adding to this one.

Reply:
![Screenshot from 2022-08-19 17-12-11](https://user-images.githubusercontent.com/77285023/185612415-31ab261c-9120-444c-8a7e-444184450572.png)
![Screenshot from 2022-08-19 17-19-13](https://user-images.githubusercontent.com/77285023/185612431-a5e55142-c518-4fc1-a10e-c7b199f1ed4f.png)